nixos/modules/system/boot/luksroot.nix

```diff
@@ -140,9 +140,12 @@ let
     umount /crypt-ramfs 2>/dev/null
   '';
 
-  openCommand = name': { name, device, header, keyFile, keyFileSize, keyFileOffset, allowDiscards, yubikey, gpgCard, fido2, fallbackToPassword, preOpenCommands, postOpenCommands,... }: assert name' == name;
+  openCommand = name': dev: with dev; assert name' == name;
```

Comment:
I would rather keep it explicit IMHO.

Reply:
I'm not sure that's necessary since this just an enumeration of the options in the `boot.initrd.luks.devices.<name>` submodule, and `openCommand` is only applied to such submodules. It seems like this line started as just `{ name, device }:`, and then people kept adding arguments without bothering to refactor it.

I could however remove `with dev;` and use explicit `dev.` everywhere.